docs/migration/fix-refs.rst

```diff
@@ -19,3 +19,10 @@
     # product.prod_config_main
     find . -type f -name '*.xml' | xargs sed -i 's/product.prod_config_main/sale.prod_config_main/g'
 
+
+``10.0-`` → ``11.0+``
+====================
+
+.. code-block:: sh
+
+    find . -type f -name '*.js' | xargs sed -i 's/core.mixins/requere("web.mixins")/g'
```

Comment:
![wrong](https://user-images.githubusercontent.com/186131/32320431-94db1588-bfdf-11e7-9004-d4049955f40e.png)

Reply:
@yelizariev 
![39-photos-of-vladimir-putin-bein](https://user-images.githubusercontent.com/15119940/32320738-91c4b772-bfe0-11e7-9bca-99c28e7f4a84.jpg)